Src/CSharpier/SyntaxTriviaExtensions.cs

```diff
@@ -4,10 +4,9 @@ public static class SyntaxTriviaExtensions
 {
     public static bool IsComment(this SyntaxTrivia syntaxTrivia)
     {
-        return syntaxTrivia.Kind()
-            is SyntaxKind.SingleLineCommentTrivia
-                or SyntaxKind.MultiLineCommentTrivia
-                or SyntaxKind.SingleLineDocumentationCommentTrivia
-                or SyntaxKind.MultiLineDocumentationCommentTrivia;
+        return syntaxTrivia.IsKind(SyntaxKind.SingleLineCommentTrivia)
```

Comment:
Looks like the vast majority of uses were along the lines of `Kind() == SyntaxKind.Something`.
Replacing these with `IsKind()` perfectly fine.

However, these 2 cases where we replace a single pattern with multiple method calls doesn't make sense to do "because performance". Looking at the code, it looks like doing `IsKind()` lets them skip a check to make sure it's a C# syntax token:
https://github.com/dotnet/roslyn/blob/dbc30912d98f02aa6ee532e89217d7f884a8fc31/src/Compilers/CSharp/Portable/CSharpExtensions.cs#L185

Since this project only deals with C# code, how do you feel about writing our own extension method that's like `Kind()` but skips the `IsCSharpKind()` check?

Reply:
I added RawSyntaxKind and then spent too much time thinking about when to use that vs using IsKind, so settled on just changing everything to use RawSyntaxKind.